src/main/java/org/jabref/gui/Dark.css

```diff
@@ -64,3 +64,16 @@
     -fx-background-color: -fx-light-text-color, -fx-control-inner-background;
 }
 
+.scroll-bar:horizontal .track,
```

Comment:
Similar code is already in the `Base.css` file:
https://github.com/JabRef/jabref/blob/5376dcfb648da7ad43606694da841b569b7bfb26/src/main/java/org/jabref/gui/Base.css#L704-L717

Thus, it should be sufficient if you overwrite the following color specifications:
https://github.com/JabRef/jabref/blob/5376dcfb648da7ad43606694da841b569b7bfb26/src/main/java/org/jabref/gui/Base.css#L89-L91
here in the `root` part of `Dark.css`.